packages/core/src/animation/animationCurve/AnimationArrayCurve.ts

```diff
@@ -11,7 +11,7 @@ import { IAnimationCurveCalculator } from "./interfaces/IAnimationCurveCalculato
 @StaticInterfaceImplement<IAnimationCurveCalculator<number[]>>()
 export class AnimationArrayCurve extends AnimationCurve<number[]> {
   /** @internal */
-  static _isReferenceType: boolean = true;
+  static _keepOriginReference: boolean = true;
```

Comment:
`_keepOriginReference` is strange, the context is `AnimationArrayCurve`, so the name should `_isCopyMode`

Reply:
done